style.css

```diff
@@ -570,6 +636,120 @@ button:disabled {
   visibility: hidden;
 }
 
+.modalcontainer {
+  background-color: #fff;
+  height: 631px;
+  width: 1156px;
+}
+
+.popup {
+  visibility: hidden;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 5%;
+  backdrop-filter: blur(5px);
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.overlay {
+  position: fixed;
+  backdrop-filter: blur(5px);
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  visibility: hidden;
+}
```

Comment:
- [ ] You have done well in styling the popup for the desktop version. However, as illustrated below, please adjust the margins and padding of certain elements, such as the title, to align with the template.

**Your design**

![Screenshot from 2023-03-16 05-58-39](https://user-images.githubusercontent.com/105117832/225597102-aee513c9-74df-4bd3-b6cc-85ab24252b20.png)

**Figma**

![Screenshot from 2023-03-16 05-58-56](https://user-images.githubusercontent.com/105117832/225597017-abbc1fbf-0e90-48f7-ab6e-34f66c42a3af.png)

- [ ] Although you did a good job creating the popup, it is not properly centered in the mobile version as required by the Figma template. Additionally, the buttons are not taking up the entire width and are instead aligned to the left.
Please ensure that you adjust the margins and padding of each element to match the Figma template. This will help to ensure consistency and adherence to the design standards.

**Your design**

![Screenshot from 2023-03-16 05-47-02](https://user-images.githubusercontent.com/105117832/225597244-c0e34af2-e45f-4296-9451-b84dd67588d5.png)

**Figma**

![Screenshot from 2023-03-16 05-48-43](https://user-images.githubusercontent.com/105117832/225597314-36feee04-f67d-4232-9610-53e4c2ee84e2.png)